docs/GettingStarted.md

```diff
@@ -72,7 +80,20 @@ Sub DoSomething()
 End Sub
 ````
 
-...with a single click.
+...with a single click!
+
+**Rubberduck > Indent > Current Procedure / Module / Project**  
+(also accessible through *RightClick > Toggle > Rubberduck*)
```

Comment:
That is not intentional.... or did you customize your context menu to do this?

Reply:
Oh interesting.. no customization, I'm seeing it in this order (will attach screenshot shortly) could be because I'm running Rubberduck on Office 365 on my Windows 10?

Reply:
hmm ... I wonder whether that's because the context menu order changed under Office 365 and we're setting that up incorrectly...

Reply:
interesting @Vogel612 ..
well, here is the screenshot of the option I am seeing
I would be happy to run any config/spec tests on my Office 365 for you guys if you want any 😄 
![image](https://user-images.githubusercontent.com/2426264/59856707-75ce6d80-9345-11e9-90d9-7240a639e448.png)

Reply:
Wow... the Rubberduck menu is definitely not intended to be buried like that. Created issue #5016 for this.